src/systemctl/systemctl-logind.c

```diff
@@ -366,17 +366,23 @@ int logind_show_shutdown(void) {
                 return log_error_errno(SYNTHETIC_ERRNO(ENODATA), "No scheduled shutdown.");
 
         if (STR_IN_SET(action, "halt", "poweroff", "exit"))
-                action = "Shutdown";
+                pretty_action = "Shutdown";
         else if (streq(action, "kexec"))
-                action = "Reboot via kexec";
+                pretty_action = "Reboot via kexec";
         else if (streq(action, "reboot"))
-                action = "Reboot";
+                pretty_action = "Reboot";
 
         /* If we don't recognize the action string, we'll show it as-is */
```

Comment:
Not a big matter, but maybe better to add the following:
```c
else /* If we don't recognize the action string, we'll show it as-is */
        pretty_action = action;
```
Then you can simply use `pretty_action` below.